azure-pipelines/build_windows.yaml

```diff
@@ -0,0 +1,12 @@
+steps:
+  - template: checkout_windows.yml
```

Comment:
Can you also keep 

 - task: ComponentGovernanceComponentDetection@0
    condition: succeeded()
    inputs:
      scanType: 'Register'
      verbosity: 'Normal'
      alertWarningLevel: 'High'
      failOnAlert: true
      failOnStderr: true

Reply:
Done